modules/apps/sharing/sharing-api/src/main/java/com/liferay/sharing/security/permission/SharingEntryAction.java

```diff
@@ -20,7 +20,8 @@
  */
 public enum SharingEntryAction {
 
-	ADD_DISCUSSION("ADD_DISCUSSION", 4), UPDATE("UPDATE", 2), VIEW("VIEW", 1);
+	ADD_DISCUSSION("ADD_DISCUSSION", 4), DOWNLOAD("DOWNLOAD", 32),
```

Comment:
Hi, as on the commit said I'm adding the DOWNLOAD permission with the bitwise of ResourceAction to maintain the coherence between the two of them.

Do you think it's better to use other?

Reply:
Since there is no connection between `SharingEntryAction` and the value stored in `ResourceAction` I would use the value `8`. Otherwise nobody checking the code would understand this gap between 4 and 32.

Reply:
<img src="https://media1.giphy.com/media/QYwB8ai7mtORGRAxJZ/giphy.gif"/>